subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BomSupportPluginsSmokeTest.groovy

```diff
@@ -64,64 +67,66 @@ class BomSupportPluginsSmokeTest extends AbstractSmokeTest {
+            module("org.springframework:spring-beans:5.0.5.RELEASE")
+            module("org.springframework:spring-context:5.0.5.RELEASE")
+            module("org.springframework:spring-core:5.0.5.RELEASE")
             module("junit:junit:4.12")
             module("org.hamcrest:hamcrest-core:1.3")
         }
         def springBootDeps = {
-            module("org.springframework:spring-core:4.3.12.RELEASE", directBomDependency ? {} : springCoreDeps).byReason(reason3)
-            module("org.springframework:spring-context:4.3.12.RELEASE", directBomDependency ? {} : springContextDeps).byReason(reason3)
-            module("org.springframework:spring-test:4.3.12.RELEASE")
+            module("org.springframework:spring-core:5.0.5.RELEASE", directBomDependency ? {} : springCoreDeps).byReason(reason3)
+            module("org.springframework:spring-context:5.0.5.RELEASE", directBomDependency ? {} : springContextDeps).byReason(reason3)
+//            module("org.springframework:spring-jcl:5.0.5.RELEASE")
+            module("org.springframework:spring-test:5.0.5.RELEASE")
             module("junit:junit:4.12")
         }
         def springBootAutoconfigureDeps = {
             module("org.springframework.boot:spring-boot:$bomVersion")
         }
         def springBootTestDeps = {
             module("org.springframework.boot:spring-boot:$bomVersion")
-            module("org.springframework:spring-test:4.3.12.RELEASE")
+            module("org.springframework:spring-test:5.0.5.RELEASE")
             module("junit:junit:4.12")
             module("org.hamcrest:hamcrest-core:1.3")
         }
         def springBootTestAutoconfigureDeps = {
             module("org.springframework.boot:spring-boot-test:$bomVersion")
             module("org.springframework.boot:spring-boot-autoconfigure:$bomVersion")
-            module("org.springframework:spring-test:4.3.12.RELEASE")
+            module("org.springframework:spring-test:5.0.5.RELEASE")
         }
 
         resolve.expectDefaultConfiguration('compile')
         resolve.expectGraph {
             root(':', ':springbootproject:') {
                 if (directBomDependency) {
                     module("org.springframework.boot:spring-boot-dependencies:$bomVersion") {
-                        module("org.springframework:spring-core:4.3.12.RELEASE", springCoreDeps)
-                        module("org.springframework:spring-aop:4.3.12.RELEASE", springAopDeps)
-                        module("org.springframework:spring-beans:4.3.12.RELEASE", springBeansDeps)
-                        module("org.springframework:spring-context:4.3.12.RELEASE", springContextDeps)
-                        module("org.springframework:spring-expression:4.3.12.RELEASE", springExpressionDeps)
-                        module("org.springframework:spring-test:4.3.12.RELEASE")
+                        module("org.springframework:spring-core:5.0.5.RELEASE", springCoreDeps)
```

Comment:
I don't use a property "5.0.5.RELEASE" because here it's evaluated in a closure which is delegated to `GraphBuilder` and `expectGraph()` complains about property not found.